tests/snap/semantic_tokens/explicit_instantiation_directives.cpp

```
@@ -0,0 +1,23 @@
+/// ## Function and variable explicit instantiation directives — clang builds no node for the directive, so every identifier on it goes unpainted: the name, the template arguments, even the declarator's type
+///
+/// - status: partial
+/// - issues: llvm#191658
+/// - order: 26
+
+struct Widget {};
+
+template <typename T>
+void convert(T value) {}
+
+extern template void §convert<§Widget>(§Widget);
```

Cppcheck (2.21.0)

[error] The code contains unhandled character(s) (character code=194). Neither unicode nor extended ascii is supported.

(unhandledChar)